### PR TITLE
Add styles

### PR DESCRIPTION
**Pull request by DanteAlonsoHT**
# 📚   Bookstore: styling 🎨 

For this repository, I've created a project to create a React project using Redux, the main objective is to learn the way to create an application to store, use actions and reducers in React, to understand the concepts about state management, connect React and Redux, connect an existing API via Redux and use the data to display something in the UI, and apply React best practices and language style guides in code.

- [x] - Update components/FormBooks.js
- [x] - Update components/Book.js
- [x] - Update  components/pages/Home.js
- [x] - Add multiple styles in index.css
- [x] - Update App.js
- [x] - Fix linters 
- [x] - Upload image to show the final result 
- [x] - Update descriptive README file

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @DanteAlonsoHT 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet) 
 
-# Bookstore: create and remove books.
+# Bookstore: React and Redux Project.
 
-For this repository, I've created a project to create a React project using Redux, the main objective is to learn the way to create an application to store, use actions and reducers in React, to understand the concepts about state management, and connect React and Redux.
+For this repository, I've created a project to create a React project using Redux, the main objective is to learn the way to create an application to store, use actions and reducers in React, to understand the concepts about state management, connect React and Redux, connect an existing API via Redux and use the data to display something in the UI, and apply React best practices and language style guides in code.
```
Comment (mikenath223) [inline README.md:5]: - [x] Please deploy the project online and include the live deploy link in the readme.

File: src/index.css [hunk 2]
```diff
@@ -1,5 +1,192 @@
...
+}
+
+nav div a {
+  text-decoration: none;
+  margin-right: 2.5rem;
+  font-weight: 600;
+  color: #aaa;
+}
+
+/* stylelint-disable-next-line selector-type-no-unknown */
```
Comment (mikenath223) [inline src/index.css]: - [x] Kindly remove this stylelint disable line its best to fix the issue in the code.

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @DanteAlonsoHT, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**